app/src/main/res/values-af-rZA/strings.xml

```diff
@@ -73,4 +73,7 @@
   <string name="version_beta">Beta</string>
   <string name="version_alpha">Alpha</string>
   <string name="version_load_fail">Failed to retrieve version</string>
+    <string name="installer_completed">Completed!</string>
+  <string name="installer_copy_text">Copy</string>
```

Comment:
formatting + newline
also remove all of the changes from the other translated files